tests/test_types.py

```diff
@@ -560,13 +560,13 @@ def test_deserialization_struct() -> None:
 
 
 def test_str_struct(simple_struct: StructType) -> None:
-    assert str(simple_struct) == "struct<1: required_field: required string (this is a doc), 2: optional_field: required int>"
+    assert str(simple_struct) == "struct<1: required_field: required string (this is a doc), 2: optional_field: optional int>"
```

Comment:
This seems a bit confusing, the name of the field is `required_field`, but it is actually optional.

Reply:
Misread this, sorry about that!

Reply:
acknowledged